packages/db/tests/lib/test-granule.js

```diff
@@ -0,0 +1,427 @@
+const test = require('ava');
```

Comment:
I did a quick look through the PR changeset diff from the previous review and this appears to be a move/test name change from the original in the granule model unit test.  👍🏻  assuming that's true.  Let me know if that's incorrect and I need to review it beyond the original review.